proc/merge/merge.go

```diff
@@ -9,6 +9,7 @@ import (
 
 // A Merge proc merges multiple upstream inputs into one output.
 type Proc struct {
+	ctx      *proc.Context
```

Comment:
@alfred-landrum: Can we change the name of this to pctx? ctx means context.Context everywhere else in the repo.